.github/workflows/behat.yaml

```diff
@@ -0,0 +1,167 @@
+            fail-fast: false
+            matrix:
+                php: [7.4]
+                sylius: ['~1.7.0']
+
+        env:
+            APP_ENV: test
+
+        steps:
+            - name: PHP - Switch
+              run: sudo update-alternatives --set php /usr/bin/php${{ matrix.php }} && php -v
+
+            - name: PHP - Setup timezone
+              run: |
+                  echo "date.timezone=UTC" >> /tmp/timezone.ini; \
+                  sudo mv /tmp/timezone.ini /etc/php/${{ matrix.php }}/cli/conf.d/timezone.ini
+
+            - uses: actions/checkout@v2
+              with:
+                  path: theme
+
+            - uses: actions/cache@v1
+              id: cache-composer
+              with:
+                  path: /home/runner/.composer/cache
+                  key: composer-php.${{ matrix.php }}-sylius.${{ matrix.sylius }}-${{ github.sha }}
+                  restore-keys: composer-php.${{ matrix.php }}-sylius.${{ matrix.sylius }}-
+
+            - name: Composer - Create cache directory
+              run: mkdir -p /home/runner/.composer/cache
+              if: steps.cache-composer.outputs.cache-hit != 'true'
+
+            - name: Composer - Self Update
+              run: |
+                  mkdir -p /home/runner/.composer/
+                  sudo composer self-update
+
+            - name: Composer - Github Auth
+              run: composer config -g github-oauth.github.com ${{ github.token }}
+
+            - name: Composer - Install Sylius-Standard
+              run: composer create-project --no-progress sylius/sylius-standard sylius "${{ matrix.sylius }}"
+
+            - name: Composer - Add path repository
+              working-directory: ./sylius
+              run: |
+                  composer config repositories.plugin '{"type": "path", "url": "../theme/"}'
+
+            - name: Theme - Install Theme
+              working-directory: ./sylius
+              run: composer require sylius/bootstrap-theme encore
+
+            - name: Theme - Copy Theme
+              working-directory: ./sylius
+              run: |
+                  mkdir -p themes/BootstrapTheme/
+                  cp -r ../theme/* themes/BootstrapTheme/
+
+            - name: Theme - Install dependencies
+              working-directory: ./sylius
+              run: |
+                  yarn
+                  yarn add @symfony/webpack-encore sass-loader@^7.0.0 node-sass lodash.throttle -D
+                  yarn add bootstrap@^4.5.0 bootstrap.native@^3.0.0 glightbox axios form-serialize @fortawesome/fontawesome-svg-core @fortawesome/free-brands-svg-icons @fortawesome/free-regular-svg-icons @fortawesome/free-solid-svg-icons
+
+            - name: Theme - Config Webpack
+              working-directory: ./sylius
+              run: |
+                  echo "const bootstrapTheme = require('./themes/BootstrapTheme/webpack.config');" >> webpack.config.js
+                  echo "module.exports = [bootstrapTheme];" >> webpack.config.js
```

Comment:
Does this not leave us with 2 "module.exports = .." lines in the `webpack.config.js` file? I think this would override the previous assignment containing the shop and admin exports.